### PR TITLE
docs(proxy) add `X-` prefix to Kong specific headers

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

File: app/docs/0.13.x/proxy.md [hunk 1]
```diff
@@ -878,10 +878,10 @@ following headers will be added by Kong and the full set of headers be sent to
 the client:
 
 - `Via: kong/x.x.x`, where `x.x.x` is the Kong version in use
-- `Kong-Proxy-Latency: <latency>`, where `latency` is the time in milliseconds
+- `X-Kong-Proxy-Latency: <latency>`, where `latency` is the time in milliseconds
   between Kong receiving the request from the client and sending the request to
   your upstream service.
-- `Kong-Upstream-Latency: <latency>`, where `latency` is the time in
+- `X-Kong-Upstream-Latency: <latency>`, where `latency` is the time in
```
Comment (thibaultcha) [inline app/docs/0.13.x/proxy.md]: @hbagdi I think this version of the patch is missing the updates for the previous docs versions?

Comment (hbagdi) [reply app/docs/0.13.x/proxy.md]: @thibaultcha I updated the previous version in #618 which was against master and this one was for `0.13.x` only
Do you want to put all of those in here?

Comment (thibaultcha) [reply app/docs/0.13.x/proxy.md]: @hbagdi Got it, sorry about that

**Comment by thibaultcha** [comment]
Thanks!

**Pull request merged**